### PR TITLE
sortMethod simplification.

### PR DESCRIPTION
**Pull request by Michael-Gallo**
The iota enum expression for sortMethod required the same lengthy switch statement in setExpression's `eval` and `setLocalExpression`'s eval, along with `getOptsMap` in main to be updated for every sort method.  Making sortMethod's underlying representation a string removes the need to have these long and repetitive switch statements and for new sortMethods to require changes outside of `dir.sort` in `nav.go` and the const definitions at the top of `opts.go`

Alternatively, `sortMethod` can stay as an iota and there can be a map of sortMethods to their string representations.

### CODE REVIEW

File: opts.go
Comment (joelim-work) [inline opts.go:22]: You forgot `timeSort`:

```suggestion
	for _, validMethod := range []sortMethod{naturalSort, nameSort, sizeSort, timeSort, atimeSort, ctimeSort, extSort} {
```

**Review by joelim-work — COMMENTED** [review]
Thanks for this patch. From what I can see in the commit history, it looks like from a0e1d4d the sort method actually used to be a string. I'm not sure the reason behind the change though, so I will leave it to @gokcehan to decide before merging.

**Review by joelim-work — APPROVED** [review]

**Comment by gokcehan** [comment]
@joelim-work I think my rationale for that commit was to have a single line comparison `if d.sortType != gOpts.sortType { ... }`, but I don't see any reason why we can't simply use strings. As a further cleanup, I'm also in favor of simply getting rid of the `sortType` struct and having everything under `gOpts` since the single line comparison is already outdated in this case. Anyway, thank you @Michael-Gallo for the patch and @joelim-work for the review.

**Pull request merged**
